packages/terra-button-group/src/ButtonGroupButton.scss

```diff
@@ -1,14 +1,55 @@
 :local {
   .button-group-button {
+    background-color: var(--terra-button-group-button-background-color, #fff);
+    border-color: var(--terra-button-group-button-border-color, #c8cacb);
     border-radius: 0;
     border-style: solid;
     border-width: 1px;
+    box-shadow: var(--terra-button-group-button-box-shadow, none);
```

Comment:
We should define ltr/rtl values for the themeable box-shadows.

Reply:
@yuderekyu Many other components do separate box-shadow into lrt and rtl so I think it makes sense to be consistent. However, I'm not quite sure when using box-shadow as the focus ring. 

@neilpfeiffer Do we anticipate that the offset-x and offset-y that cast the box-shadow for buttons in a button group (or just buttons in general) be something other than `0 0` when using box-shadow as the focus ring? Consumers can set what they want for these box-shadow themeable variables. 

Do we always break up box-shadow into two themeable variables for bi-di across the board or do we make that determination case by case?